enclave/asym_keys.c

```diff
@@ -0,0 +1,553 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <openenclave/bits/safecrt.h>
+#include <openenclave/enclave.h>
+#include <openenclave/internal/ec.h>
+#include <openenclave/internal/enclavelibc.h>
+#include <openenclave/internal/kdf.h>
+#include <openenclave/internal/raise.h>
+#include <openenclave/internal/utils.h>
+
+static inline oe_result_t _check_asymmetric_key_params(
+    const oe_asymmetric_key_params_t* key_params)
+{
+    if (key_params == NULL)
+        return OE_INVALID_PARAMETER;
+
+    if (key_params->type != OE_ASYMMETRIC_KEY_EC_SECP256P1)
+        return OE_INVALID_PARAMETER;
+
+    if (key_params->format != OE_ASYMMETRIC_KEY_PEM)
+        return OE_INVALID_PARAMETER;
+
+    return OE_OK;
+}
+
+static oe_result_t _load_seal_key_by_policy(
+    oe_seal_policy_t policy,
+    uint8_t** key_buffer,
+    size_t* key_buffer_size,
+    uint8_t** key_info,
+    size_t* key_info_size)
+{
```

Comment:
key_info and key_info_size do not see to be something useful. I suggest remove those two parameter from this  API